tests/rados/test_pool_osd_recovery.py

```diff
@@ -40,312 +41,409 @@ def run(ceph_cluster, **kw) -> int:
     config = kw["config"]
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     rados_obj = RadosOrchestrator(node=cephadm)
-    pool_configs = config["pool_configs"]
-    pool_configs_path = config["pool_configs_path"]
+    pool_configs = config.get("pool_configs")
```

Comment:
The intension of giving square brackets instead of get was, so that if we do not have those keys in the config, test should fail there itself with keyerror. now it will contain None, and might fail somewhere further saying that none type is not operable.

not a issue, but yeah.

Reply:
I included the Scenario "7. Replacement of a failed OSD host" in this test module itself, but the yaml test config doesn't need pool_configs and pool_configs_path, and therefore are not identified.
Consequently, the test execution fails because of square bracket as both the inputs become necessary to proceed.
We can keep the square brackets, I will create a new test case file for the concerned scenario(Replacement of a failed OSD host)

Reply:
Not necessary. if it works, then let's keep it that way itself and complete automation.